src/client/component/gameplay.cpp

```diff
@@ -266,6 +266,39 @@ namespace gameplay
 				vel_out[2] = new_z * length_scale;
 			}
 		}
+
+
+
+		void* pm_can_start_sprint_stub()
+		{
+			return utils::hook::assemble([=](utils::hook::assembler& a)
+				{
```

Comment:
fix bracket not being formatted right (just backspace it all the way to beginning, and tab it)